frontend/package.json

```diff
@@ -39,6 +39,7 @@
     "test:integration-ci": "npx concurrently -k -s first -n \"SB,TEST\" \"npm run build:storybook --quiet && npx http-server storybook-static --port 6006 --silent\" \"npx wait-on tcp:127.0.0.1:6006 && npm run test:accessibility && npm run test:integration\"",
     "test:e2e": "export E2E=1; playwright test ./src/__tests__/e2e",
     "storybook": "storybook dev -p 6006 -s ../config",
+    "cypress:ci": "npx concurrently -k -s first \"npm run cypress:server:build && npm run cypress:server\" \"npx wait-on tcp:127.0.0.1:9001 && npm run cypress:run:mock\"",
```

Comment:
Given the pattern in our test scripts where they all start with `test:`, should we also rename this to `test:cypress-ci` for consistency?

Reply:
updated